cpp/src/Ice/ios/iAPEndpointI.mm

```diff
@@ -303,19 +303,23 @@ ICEIAP_API void registerIceIAP(bool loadOnInitialize)
 }
 
 vector<EndpointIPtr>
-IceObjC::iAPEndpointI::expandIfWildcard() const
+IceObjC::iAPEndpointI::expandHost() const
 {
```

Comment:
Would be nicer to use C++ 17 vector initialization

```
return vector<EndpointIPtr> { ... }
```

Reply:
Fixed as suggested. Also for IceBT and Opaque